configs/common/modules_tcl.yaml

```diff
@@ -35,18 +34,21 @@ modules:
       - freetype
       - gcc-runtime
       - gdbm
+      - gmake
```

Comment:
Why is gmake blacklisted? This one I could sort of imagine developers wanting to access.

Reply:
For one, if you blacklist something the modules that need it have the necessary paths in their own modulefiles. And also, even if not there is always a gmake on the system and in the default path. We need to exclude all modules from being generated that are created with a compiler different than the preferred compiler, or our module logic won't work.